packages/block-library/src/site-title/index.php

```diff
@@ -14,10 +14,15 @@
  */
 function render_block_core_site_title( $attributes ) {
 	$tag_name = 'h1';
+	$class = '';
+
 	if ( isset( $attributes['level'] ) ) {
 		$tag_name = 0 === $attributes['level'] ? 'p' : 'h' . $attributes['level'];
 	}
-	return sprintf( '<%1$s>%2$s</%1$s>', $tag_name, get_bloginfo( 'name' ) );
+	if ( isset( $attributes['align'] ) ) {
+		$class = ' class="has-text-align-' . $attributes['align'] . '"';
```

Comment:
We should append individual class names through `sprintf` and escape them.

See the Calendar block for an example.